### DISCUSSION TITLE
Cluster manager for Apache Kafka

### DISCUSSION THREAD

**Discussion by sreejesh123** [strimzi/strimzi-kafka-operator#5344] [Q&A]
we currently use  open source CMAK  for kluster management for apache kafka, what is the best recommendation for Strimzi.  

Operator is suppose to be used for cluster management, but what UI option we have? Any document you can share on this

**Comment by scholzj** [comment]
I guess it depends what you call cluster management. Strimzi currently does not have its own UI. You should be able to use one of the many available Kafka UIs out there. I think there are only two things to note:
* Strimzi by default does not offer ZooKeeper access - some old UIs might still need it and might not work because of it
* Some actions done via the UI might conflict with the operator. For example Kafka configuration changes should be always done in he `Kafka` CR and not the UI. Other example is that if you use the User Operator to manage you should use only the `KafkaUSer` CRs and not try to for example manage the ACLs directly inside Kafka from the UI. In both cases, the operator would overwrite the changes.

But most other features should be IMHO without any problems ... dealing with topics, reading messages, monitoring etc.

**Reply by sreejesh123** [reply]
thank you,  

will you be able to share any example  on how RBAC and ACL are done using CRs?  I am assuming these till be yaml which then get executed via kubectl? 

Strimzi considering an UI feature any time soon?  Though without that itself we can get a lot done than vanilla Apache Kafka.

**Reply by scholzj** [reply]
> Strimzi considering an UI feature any time soon? Though without that itself we can get a lot done than vanilla Apache Kafka.

Having a UI would be nice ... but it is definitely not going to happen anytime soon.

> will you be able to share any example on how RBAC and ACL are done using CRs? I am assuming these till be yaml which then get executed via kubectl?



ACLs can be managed in the `KafkaUser` resource. There are examples in the `examples` folder on GitHub. There is also a lot of stuff abotu it in the documentation.
